src/components/constants.ts

```diff
@@ -10,5 +13,16 @@ export const DEFAULT_PURPOSE_TO_MESSAGE_KEY: Record<string, DefinedMessage> = {
   SaleOfInfo: completeOptionsMessages.saleOfInfoLabel,
 };
 
+export const DEFAULT_PURPOSE_TO_INVERTED_MESSAGE_KEY: Record<
+  string,
+  DefinedMessage
+> = {
+  Essential: completeOptionsMessages.essentialLabel,
```

Comment:
should this be

```suggestion
  Essential: completeOptionsInvertedMessages.essentialLabel,
```

Reply:
no that's not a thing

Reply:
(i know it looks like it should be, but I checked)